contracts/src/v0.8/feeds/PrimaryAggregator.sol

```diff
@@ -457,6 +463,73 @@ contract PrimaryAggregator is SiameseAggregatorBase, OCR2Abstract, OwnerIsCreato
     return latestAggregatorRoundId + 1;
   }
 
+  /**
+   *
+   * Section: Secondary Proxy
+   *
+   */
+
+  struct Report {
+    int192 juelsPerFeeCoin;
+    uint32 observationsTimestamp;
+    bytes observers; // ith element is the index of the ith observer
+    int192[] observations; // ith element is the ith observation
+  }
+
+  struct Transmission {
+    int192 answer;
+    uint32 observationsTimestamp;
+    uint32 recordedTimestamp;
+  }
+
+  mapping(uint32 /* aggregator round ID */ => Transmission) internal s_transmissions;
+
+  address internal immutable s_secondaryProxy;
+
+  uint32 internal s_cutoffTime;
+
+  /**
+   * @notice emitted when a new cutoxx time is set
+   */
+  event CutoffTimeSet(uint64 old, uint64 current);
```

Comment:
1. Don't think old is required. If you want to include old you should rename to CutoffTimeUpdated
2. Shouldn't these values be uint32?